tests/test_python/code/README.mkd

```diff
@@ -0,0 +1,12 @@
+Hello World
```

Comment:
I'm not quite sure which unnecessary files you're referring to, but this file is useful mainly for ensuring that only Python files are analyzed. On some of my work branching off of this I ran into SyntaxErrors because I wasn't properly filtering, and this helped me catch that during manual testing.